ScanbotSDKExamples/MAUI/ClassicComponent.Maui/Handlers/BarcodeCameraViewHandler.iOS.cs

```diff
@@ -60,25 +60,14 @@ public static void MapStopDetectionHandler(BarcodeCameraViewHandler current, Bar
 
         public static void MapOnPauseHandler(BarcodeCameraViewHandler current, BarcodeCameraView commonView, object arg3)
         {
-
+            // android only method.
```

Comment:
@scan-mayank, android only in an iOS file, is that correct?

Reply:
There are common methods created in the common control(BarcodeCameraView), which have to be implemented on all available platforms.
These 2 methods `MapOnPauseHandler` and `MapOnResumeHandler` are only useful in the Android part of the common control.

Please ref BarcodeCameraView.cs, BarcodeCameraViewHandler.cs for more details

Reply:
That makes sense. It might be useful to add what you described as a comment somewhere in the project (or the file) so that it is clearer for the presence of everything, especially for a fresh user of the code.